src/System.Numerics.Vectors/src/System/Numerics/VectorMath.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#if HAS_INTRINSICS
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace System.Numerics
+{
+    internal static class VectorMath
+    {
+        public static Vector128<float> Lerp(Vector128<float> a, Vector128<float> b, Vector128<float> t)
+        {
+            if (Sse.IsSupported)
```

Comment:
It's probably sufficient to just `Debug.Assert(Sse.IsSupported)`, since this is internal and we have already checked the code higher up.